src/devtoolsPanel.ts

```diff
@@ -51,7 +51,7 @@ export class DevToolsPanel {
         // Hook up the socket events
         this.panelSocket = new PanelSocket(this.targetUrl, (e, msg) => this.postToDevTools(e, msg));
         this.panelSocket.on("ready", () => this.onSocketReady());
-        this.panelSocket.on("websocket", () => this.onSocketMessage());
+        this.panelSocket.on("websocket", (msg) => this.onSocketMessage(msg));

```

Comment:
Let's add a real closed event for the panelSocket rather than overloading the message event.

Reply:
Make sure to add a new test with these changes